components/freertos/Kconfig

```diff
@@ -421,7 +421,7 @@ menu "FreeRTOS"
 
     config FREERTOS_TASK_FUNCTION_WRAPPER
         bool "Enclose all task functions in a wrapper function"
-        depends on COMPILER_OPTIMIZATION_DEFAULT
+        depends on COMPILER_OPTIMIZATION_DEFAULT || CONFIG_ESP_COREDUMP_ENABLE || CONFIG_ESP_GDBSTUB_ENABLED
```

Comment:
Sorry, this should be `|| ESP_COREDUMP_ENABLE || ESP_GDBSTUB_ENABLED` — inside Kconfig files all options are referenced without `CONFIG_` prefix.